test/unit/mocks/mongoConnector.ts

```diff
@@ -0,0 +1,12 @@
+import { DocumentQuery, Model } from "mongoose";
+
+import { IUserModel } from "../../../src/connectors/mongodb/user";
+
+export class FakeMongoConnector {
+  public users = {
+    findById: (_) => Promise.resolve<IUserModel>(undefined) as any as DocumentQuery<IUserModel, IUserModel>,
```

Comment:
To make the TypeScript compiler happy. It is the only method on `Model` that do not have an empty parameter list as an overload.